tests/main.rs

```diff
@@ -61,7 +61,7 @@ fn it_should_watch_a_file_async() {
 
     use futures::Stream;
     let events = inotify
-        .event_stream(&mut buffer)
+        .event_stream(&mut buffer[..])
```

Comment:
I assume this change was required to get it to compile? If so, this is a breaking change. Not a problem, just want to confirm.

Reply:
Yeah, that's correct --- it's only necessary when the array is longer than `[T; 32]`, as only arrays of length 0-32 have `AsRef`/`AsMut` impls.